userspace/libsinsp/sinsp.cpp

```diff
@@ -1623,6 +1623,19 @@ bool sinsp::suppress_events_comm(const std::string &comm)
 	return true;
 }
 
+bool sinsp::suppress_events_tid(int64_t tid)
+{
+	if(m_h)
+	{
+		if(scap_suppress_events_tid(m_h, tid) != SCAP_SUCCESS)
+		{
+			return false;
+		}
+	}
+
+	return true;
```

Comment:
Do we want to return "ok" when `m_h` is still unitialized?

Reply:
You're right, that might be confusing